src/betterproto/plugin/parser.py

```diff
@@ -124,9 +124,10 @@ def generate_code(request: CodeGeneratorRequest) -> CodeGeneratorResponse:
 
     # Make each output directory a package with __init__ file
     init_files = {
-        directory.joinpath("__init__.py")
+        directory / "__init__.py"
```

Comment:
```suggestion
        directory.joinpath("__init__.py")
```

Nitpicky, but might be better to leave this as it was to keep the change minimal

Reply:
That's probably a good idea